src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString.split(';');
+
+  const stylesObj = stylesArray.reduce((acc, style) => {
+    const [key, value] = style.split(':');
```

Comment:
There's a potential issue here if the style string contains multiple colons, as it would split the value incorrectly. Although handling this case is not required, it would be good practice to consider using a limit in the split function to avoid such issues.